turbine/src/retransmit_stage.rs

```diff
@@ -412,6 +420,7 @@ pub fn retransmitter(
                 &mut shred_deduper,
                 &max_slots,
                 rpc_subscriptions.as_deref(),
+                slot_status_notifier.clone(),
```

Comment:
It would be nice if we could pass by reference instead of having to clone this in multiple places

Reply:
changed to pass reference